tests/test_object_detection.py

```diff
@@ -0,0 +1,406 @@
+            [260.174, 108.24, 273.084, 155.805, 0.056],
+        ],
+        [],
+    ],
+]
+labels = [
+    {
+        "bboxes": [
+            [388.6600036621094, 69.91999816894531, 498.07000732421875, 347.5400085449219],
+            [0.0, 262.80999755859375, 62.15999984741211, 299.5799865722656],
+            [119.4000015258789, 272.510009765625, 144.22000122070312, 306.760009765625],
+            [141.47000122070312, 267.9100036621094, 173.66000366210938, 303.7699890136719],
+        ],
+        "labels": [3, 3, 2, 2],
+        "seg_map": "000000397133.png",
+    },
+    {
+        "bboxes": [
+            [26.5, 215.25, 88.0, 229.75],
+            [116.5, 189.57000732421875, 166.5, 215.07000732421875],
+            [241.9499969482422, 180.4199981689453, 293.32000732421875, 225.82000732421875],
+        ],
+        "labels": [1, 1, 1],
+        "seg_map": "000000037777.png",
+    },
+    {
+        "bboxes": [
+            [326.2799987792969, 174.55999755859375, 397.5199890136719, 371.80999755859375],
+            [9.789999961853027, 167.05999755859375, 131.72999572753906, 393.510009765625],
+            [510.44000244140625, 171.27000427246094, 634.0999755859375, 387.0299987792969],
+            [345.1300048828125, 226.41000366210938, 356.19000244140625, 248.5500030517578],
+            [337.05999755859375, 44.11000061035156, 398.4200134277344, 101.27999877929688],
+        ],
+        "labels": [3, 3, 3, 2, 4],
+        "seg_map": "000000252219.png",
+    },
+    {
+        "bboxes": [[167.3800048828125, 293.55999755859375, 354.0799865722656, 492.05999755859375]],
+        "labels": [1],
+        "seg_map": "000000491497.png",
+    },
+    {
+        "bboxes": [
+            [567.8200073242188, 273.1000061035156, 599.2000122070312, 347.2099914550781],
+            [251.19000244140625, 106.41999816894531, 274.510009765625, 168.13999938964844],
+        ],
+        "labels": [3, 3],
+        "seg_map": "000000348881.png",
+    },
+]
+
+
+def make_numpy(labels, predictions):
+    np_labels = []
+    for ann in labels:
+        np_labels.append(
+            {
+                "bboxes": np.array(ann["bboxes"]),
+                "labels": np.array(ann["labels"]),
+                "seg_map": ann["seg_map"],
+            }
+        )
+
+    np_predictions = predictions[:]
+    np_predictions = [np.array(pred, dtype=object) for pred in np_predictions]
+    for i in range(len(np_predictions)):
+        for j in range(len(np_predictions[i])):
+            if len(np_predictions[i][j]) == 0:
+                np_predictions[i][j] = np.zeros((0, 5))
+            else:
+                np_predictions[i][j] = np.array(np_predictions[i][j])
+    return np_labels, np_predictions
+
+
+labels, predictions = make_numpy(labels, predictions)
+
+
+def test_get_label_quality_scores():
+    scores = get_label_quality_scores(labels, predictions)
+    assert len(scores) == len(labels)
+    assert (scores <= 1.0).all()
+    assert len(scores.shape) == 1
+
```

Comment:
need to assert the scores are actually correct. Ie. prediction that does not match label gets lower score than prediction that does